### PR TITLE
feat(broker): add connections_count parameter in form

### PR DESCRIPTION
**Pull request by loiclau**
# Pull Request Template

## Description

In Centreon Broker SQL connector configuration page, add an entry for the connections_count variable.
This variable is a positive integer.
Default is 1.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by cgagnaire** [comment]
Related to https://github.com/centreon/centreon-broker/commit/15f64791570e9b2983288eb245bdca0b3a271a55#diff-735e0e98dfa6d5431f148e58d3a6d8c3

File: www/include/common/javascript/centreon/brokerConfig.js [hunk 1]
```diff
@@ -140,6 +140,36 @@ jQuery(function () {
...
+// Hooks for some fields
+var countConnections = {
+    // Hook on load tab
+    onLoad: function (element, argument) {
+        argument = window.JSON.parse(argument);
+        return function () {
+            var entry = element.name.match('(input|output)(\\[\\d\\])\\[(\\w*)\\]');
+            var target = entry[1] + entry[2] + '[' + argument.target + ']';
+
+            if(document.getElementsByName(target)[1].value == '') {
```
Comment (kduret) [inline www/include/common/javascript/centreon/brokerConfig.js]:
```suggestion
            if (document.getElementsByName(target)[1].value == '') {
```

File: www/install/php/Update-20.04.0-beta.1.php [hunk 2]
```diff
@@ -89,3 +89,40 @@
...
+
+
+// Change broker sql output form
+
+try {
+    // reorganise existing input form
+    $pearDB->query(
+        "UPDATE cb_type_field_relation AS A INNER JOIN cb_type_field_relation AS B ON A.cb_type_id = B.cb_type_id
+        SET A.`order_display` = 8 
+        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname ='buffering_timeout')"
```
Comment (kduret) [inline www/install/php/Update-20.04.0-beta.1.php]:
```suggestion
        WHERE B.`cb_field_id` = (SELECT f.cb_field_id FROM cb_field f WHERE f.fieldname = 'buffering_timeout')"
```

File: www/install/php/Update-20.04.0-beta.1.php [hunk 3]
```diff
@@ -89,3 +89,39 @@
     );
     throw new \PDOException($e);
 }
+
+
```
Comment (sc979) [inline www/install/php/Update-20.04.0-beta.1.php]:
```suggestion
```

File: www/install/php/Update-20.04.0-beta.1.php [hunk 4]
```diff
@@ -89,3 +89,39 @@
...
+
```
Comment (sc979) [inline www/install/php/Update-20.04.0-beta.1.php]:
```suggestion
```

**Pull request merged**